src/components/playback/playbackmanager.js

```diff
@@ -2969,6 +2970,7 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
             } else {
                 self._playQueueManager.queue(items);
             }
+            events.trigger(player, 'playlistitemmove');
```

Comment:
By meaning, `playlistitemadd` fits better. But it isn't handled.

Reply:
@dmitrylyzo Didn't know that we had an event for that and didn't want to mess so much with them. I checked and added it to ``remotecontrol.js```

Good catch!